10_files_lab/fileslab.txt

```diff
@@ -9,29 +9,65 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
 
 Write here the commands you used:
 
+mkdir lab
+cd lab/
+touch main.c game.c enemy.chero.ca.out
+touch monster.h human.h
+touch .highscore
+
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
 Write here the commands you used:
 
+cd lab/
+mkdir music misc drivers
+
```

Comment:
Notice in Unix files/directory names are case sensitive, so `mkdir music` is not the same as `mkdir Music`.